src/main/resources/io/github/dgroup/arch4u/pmd/arch4u-template-ruleset.xml

```diff
@@ -77,4 +78,33 @@
       ]]>
     </example>
   </rule>
+
+  <rule name="AvoidProhibitedMethodInvocation"
+        since="0.4.0"
+        language="java"
+        message="The {0}#{1} invocation is prohibited to use."
+        class="io.github.dgroup.arch4u.pmd.AvoidProhibitedMethodInvocation">
+    <description>
+      Avoid to use prohibited method invocation.
+      See discussion about tips/tricks implementation on Github:
+      https://github.com/pmd/pmd/discussions/TBD
```

Comment:
Avoid using the prohibited method(s).
See discussion about implementation tips/tricks on Github:
https://github.com/dgroup/arch4u-pmd/discussions/40